addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/environments-sc/ScEnvironmentCard.js

```diff
@@ -176,6 +177,21 @@ class ScEnvironmentCard extends React.Component {
     );
   }
 
+  renderLegacyRStudioWarning(env) {
+    const metaConnection1Type = env.outputs.find(obj => obj.OutputKey === 'MetaConnection1Type');
+    if (metaConnection1Type && metaConnection1Type.OutputValue.toLowerCase() === 'rstudio' && env.state.canTerminate) {
+      return (
+        <Message
+          icon="warning"
+          header="Legacy RStudio environment found"
+          content="Please terminate this workspace as soon as possible. Support for this environment type has been deprecated. Please use RStudioV2 instead."
```

Comment:
Anything else we can point to here to help out. For instance link to our documentation?